test/unit/instance/build.test.js

```diff
@@ -71,5 +71,21 @@ describe(Support.getTestDialectTeaser('Instance'), function() {
       expect(instance.get('id')).not.to.be.undefined;
       expect(instance.get('id')).to.be.ok;
     });
+
+    it('should change values for individual instances only', function () {
+      var Model = current.define('Model', {
+        model_attribute: {
+          type: DataTypes.JSONB,
+          defaultValue: { key: 'default value' }
+        }
+      })
```

Comment:
Here, I'm following the convention of this file (see e.g. test above). I believe it's better to follow that convention, specially if we plan to submit this upstream later.